featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -238,7 +328,13 @@ def pd_avg_time_between(x):
 
 
 class Median(AggregationPrimitive):
-    """Finds the median value of any feature with well-ordered values."""
+    """Determines the middlemost number in a list of values (sorted).
```

Comment:
not sure we need the sorted part at the end. i think it's fine without it

Reply:
Removed.